apps/challenges/challenge_config_utils.py

```diff
@@ -451,10 +476,13 @@ def validate_challenge_config_util(
     phase_codenames = []
     files["challenge_test_annotation_files"] = []
     for data in challenge_phases_data:
-        if data["codename"] not in phase_codenames:
-            phase_codenames.append(data["codename"])
+        if "codename" not in data:
+            error_messages.append("ERROR: No codename found. Please add codename and try again!")
```

Comment:
No codename for what? Also keep the messages consistent between dataset split and challenge phases.